src/pages/Products/Card/Card.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './Card.scss';
+// import { Link } from 'react-router-dom';
+
+class Card extends Component {
+  render() {
+    // const { img, alt, title, hashtag, price, tag, id } = this.props;
+
+    const { list } = this.props;
+    return (
+      <li className="card" data-num={list.id}>
```

Comment:
- 주석처리된 내용들 지금 필요하지 않으면 코드에서 제거해주세요
- 구분되는 요소들은 한줄 띄워 구분해주세요
```suggestion

class Card extends Component {
  render() {
    const { list } = this.props;
    
    return (
      <li className="card" data-num={list.id}>
```